lib/board.rb

```diff
@@ -0,0 +1,25 @@
+require_relative './cell'
+class Board
+ attr_reader  :cells
+  def initialize
+    @cells = Hash.new
+
+  end
+
+  def generate_cells
+    numbers = ("1".."4").to_a
+    letters = ("A".."D").to_a
+
+    local_coordinates = []
```

Comment:
On Slack call - Arique realized this could be a 'helper' method.
working on refactoring together

Reply:
:). Thank you for talking through this with me!